src/coreclr/vm/eetwain.cpp

```diff
@@ -430,7 +430,7 @@ HRESULT EECodeManager::FixContextForEnC(PCONTEXT         pCtx,
             {
                 // This is an explicit (not special) var, so add its varNumber + 1 to our
                 // max count ("+1" because varNumber is zero-based).
-                oldNumVars = max(oldNumVars, unsigned(-ICorDebugInfo::UNKNOWN_ILNUM) + varNumber + 1);
+                oldNumVars = max(oldNumVars, (unsigned int)(unsigned(-ICorDebugInfo::UNKNOWN_ILNUM) + varNumber + 1));
```

Comment:
We are mixing `unsigned int` and `unsigned`. If I recall, `unsigned` implies `int`, why the inconsistency?

Reply:
Same below.